openapi_python_client/templates/property_templates/datetime_property.pyi

```diff
@@ -14,7 +14,7 @@ if _{{ property.python_name }} is not None:
 {% endif %}
 {% endmacro %}
 
-{% macro transform(property, source, destination, declare_type=True) %}
+{% macro transform(property, source, destination, declare_type=True, query_parameter=False) %}
 {% if property.required %}
 {% if property.nullable %}
 {{ destination }} = {{ source }}.isoformat() if {{ source }} else None
```

Comment:
There's probably something in here too. I guess we need some end to end tests with just nullable or just non-required to make sure we're covering those bases? We don't want to ever pass None to a query param. I'm going to stop writing this same note over and over now 😅

Reply:
I added every permutation of `nullable, required` for the `datetime` query parameter to the E2E test.

However, I wasn't sure how far to take it--ideally it would cover all types, but that would be an exponential explosion 😬